py/newsletter.py

```diff
@@ -30,12 +30,12 @@
             _head()
         with tr().add(td()):
             _greetings(greet_title)
-        with tr().add(td()):
-            _bodylineleft()
-        with tr().add(td()):
-            _bodylineright()
-        with tr().add(td()):
-            _bodynoline()
+        # with tr().add(td()):
```

Comment:
remove these comments again.

pro tip: make sure you look over the files changed in your PR before you (re)request

Reply:
Ah yeah I'll keep that in mind thanks!